lighthouse-core/test/fixtures/config-helpers/lighthouse-plugin-config-helper/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can you rename this folder to just `config` (instead of `config-helpers`)?

Reply:
OK. I'll do it!